scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/example/nnframes/xgboost/README.md

```diff
@@ -46,4 +48,43 @@ parameters:
 - num_round : Int 
 - path_to_model_saved : String
 
-note: make sure num_threads is larger than spark.task.cpus.
+**note: make sure num_threads is larger than spark.task.cpus.**
+
+# XGBoostClassifier Predict Example
+## Run:
+```
+spark-submit \
+  --master local[4] \
+  --conf spark.task.cpus=2 \
+  --class com.intel.analytics.bigdl.dllib.examples.nnframes.xgboost.xgbClassifierPredictExample \
+  /path/to/bigdl-dllib-spark_2.4.6-0.14.0-SNAPSHOT-jar-with-dependencies.jar \
+  /path/to/iris.data 2 100 /path/to/model/saved
+```
```

Comment:
It would be great if we can add comments for the parameters 2 and 100

Reply:
From the prediction code, seems like we only need the data path and model path as the input parameters?

Reply:
I gave a explanation:

parameters:
* path_to_iris.data : String
* num_threads : Int
* num_round : Int
* path_to_model_saved : String
Do we need a more concrete?

Reply:
I think that ok. Do we have similar explainations for predict example?

Reply:
No. I will add one in a new pr and delete args `2` and `100` in predict command.

Reply:
New [pr](https://github.com/intel-analytics/BigDL/pull/3703).